asab/api/doc.py

```diff
@@ -63,19 +153,58 @@ def build_swagger_specs(self):
 					"name": "ASAB microservice",
 					"url": "https://www.github.com/teskalabs/asab",
 				},
-				"version": "1.0.0"
+				"version": "1.0.0",
 			},
 			"servers": [{"url": "../../"}],  # Base path relative to openapi endpoint
 			"paths": {},
-
 			# Authorization
 			# TODO: Authorization must not be always of OAuth type
 			"components": {},
 		}
 
-		# Get rid of securitySchemes if there is no authorizationUrl or tokenUrl
+	def get_description(self, docstring: str | None) -> str:
```

Comment:
this can actually be a function, not a method. it does not need the DocHandler reference. you can move this outside the DocHandler class, to the end of the file.

also check your other methods. if there is no `self` needed, make them a standalone function.